sal-common/src/main/java/org/ow2/proactive/sal/model/CloudConfiguration.java

```diff
@@ -6,105 +6,59 @@
 package org.ow2.proactive.sal.model;
 
 import java.io.Serializable;
+import java.util.LinkedHashMap;
 import java.util.Map;
-import java.util.Objects;
 
 import javax.persistence.*;
 
+import org.ow2.proactive.sal.util.ModelUtils;
+
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.Setter;
 
 
 /**
- * Repesents the configuration of a cloud.
+ * Represents the configuration of a cloud.
  */
 @AllArgsConstructor
 @NoArgsConstructor
 @Embeddable
+@Getter
+@Setter
+@EqualsAndHashCode(of = { CloudConfiguration.JSON_NODE_GROUP, CloudConfiguration.JSON_PROPERTIES })
```

Comment:
Looks like `@EqualsAndHashCode` is enough as nodeGroup and properties are the only two fields to compare